{{cookiecutter.repo_name}}/readthedocs.yaml

```diff
@@ -0,0 +1,16 @@
+# readthedocs.yaml
+
+version: 2
+
+build:
+  os: ubuntu-20.04
```

Comment:
do they let you do latest here? this would lower our maintenance burdens

Reply:
I swear I used to be able to, but I think we're restricted now to 20.04 or 22.04 https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os

Reply:
that's a pain, could we use 22.04 here at least so we get 4 years of blissfully not having to think about it?